cfme/tests/containers/test_containers_default_views.py

```diff
@@ -16,22 +16,22 @@
 
 
 VIEWS = ['Grid View', 'Tile View', 'List View']
-BUTTON_GROUP = ['Containers Providers', 'Projects', 'Routes', 'Nodes', 'Containers', 'Replicators']
 
-# TODO: Replace with navigator registered destinations when all container objects support them
+
+# Note: Cannot use navmazing destinations due to resetter (we need the default views)
 mapping = {
     'Containers Providers': 'containers_providers',
     'Projects': 'containers_projects',
     'Routes': 'containers_routes',
-    # 'Nodes': 'containers_nodes',
+    'Nodes': 'containers_nodes',
```

Comment:
I thought this force_navigate destionation was removed and that was why it was commented out. Please advise.